filebeat/tests/system/test_prospector.py

```diff
@@ -177,8 +177,8 @@ def test_exclude_files(self):
     def test_rotating_ignore_older_low_write_rate(self):
         self.render_config_template(
             path=os.path.abspath(self.working_dir) + "/log/*",
-            ignoreOlder="1s",
-            closeOlder="1s",
+            ignoreOlder="10s",
+            closeOlder="2s",
```

Comment:
It is intentional, as in a previous version of the change the file was sometimes closed too fast. I will reset it to 1s again as I think with the most recent changes filebeat should be faster again with rotating files. But I renamed the tests to close_older instead of ignore_older as the name is not accurate anymore. It tests close_older. Will push a new commit.